content/02.introduction.md

```diff
@@ -1,29 +1,25 @@
-Over the past decade preprints have made a resurgence within the life sciences community [@doi:10.1101/833400; @doi:10.7554/eLife.45133.001].
-Preprints are now becoming widely accepted and used within the life sciences and other communities [@doi:10.1038/530265a; @doi:10.1016/j.bpj.2016.06.035; @doi:10.1126/science.aaf9133; @doi:10.1038/s41557-020-0477-5; @doi:10.1371/journal.pmed.1002549; @doi:10.1038/530265a; @doi:10.1371/journal.pbio.3000151; @doi:10.1099/mgen.0.000259].
-Common preprint repositories include arXiv [@doi:10.1108/14666180010345564], bioRxiv [@doi:10.1101/833400] and medRxiv [@url:https://www.medrxiv.org/]; however, there are over 60 different repositories available [@url:https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/].
-
-The scientific community has begun to analyze the impact of preprints in the life sciences.
-Preprints are being posted at an increasing rate [@doi:10.1371/journal.pbio.3000269].
-Preprints are also rapidly shared on social media, routinely downloaded, and cited [@doi:10.1371/journal.pone.0047523].
-Articles with matching preprint versions are cited and discussed more often than articles without them [@doi:10.1162/qss_a_00043; @doi:10.7554/eLife.52646].
+Preprints are now hosted on online repositories, where users have open and easy access to these works.
+Notable repositories include arXiv [@doi:10.1108/14666180010345564], bioRxiv [@doi:10.1101/833400] and medRxiv [@url:https://www.medrxiv.org/]; however, there are over 60 different repositories available [@url:https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/].
+Previous work that focused on the bioRxiv repository found that life science preprints are being posted at an increase rate [@doi:10.1371/journal.pbio.3000269].
+Furthermore, these preprints are also being rapidly shared on social media, routinely downloaded, and cited [@doi:10.1371/journal.pone.0047523].
 Certain categories of preprints seem to be read and shared differently by both scientists and non-scientists [@doi:10.1101/2020.03.06.981589].
-Across preprint servers, analyses suggest that between two-thirds to three-quarters of preprints are eventually published [@doi:10.7554/eLife.45133.001; @doi:10.1002/pra2.175].
-The time required for a preprint to be published can vary from preprint to preprint; however, preprints with a single version often take less time to publish than preprints with multiple versions [@doi:10.2139/ssrn.3455146], suggesting that authors may update their preprints between submissions for peer review.
+Plus, life science articles with matching preprint versions are cited and discussed more often than articles without them [@doi:10.1162/qss_a_00043; @doi:10.7554/eLife.52646].
+These studies have primarily focused on the metadata behind preprints; however, the textual content contained within preprints has yet to be examined.

```

Comment:
```suggestion
Studies of the modern life sciences preprint repositories have primarily focused on the metadata associated with preprints; however, the textual content contained within preprints in these repositories has yet to be examined.
```
I think restricted to the domain of modern life sciences preprint repositories this is true. You should do a literature search to confirm.